src/cascadia/UnitTests_Control/ControlCoreTests.cpp

```diff
@@ -19,13 +22,185 @@ namespace ControlUnitTests
         BEGIN_TEST_CLASS(ControlCoreTests)
         END_TEST_CLASS()
 
-        TEST_METHOD(PlaceholderTest);
+        TEST_METHOD(OnStackSettings);
+        TEST_METHOD(ComPtrSettings);
+        TEST_METHOD(InstantiateCore);
+        TEST_METHOD(TestInitialize);
+        TEST_METHOD(TestAdjustAcrylic);
+
+        TEST_METHOD(TestFreeAfterClose);
+
+        TEST_METHOD(TestFontInitializedInCtor);
+
+        TEST_CLASS_SETUP(ModuleSetup)
+        {
+            winrt::init_apartment(winrt::apartment_type::single_threaded);
+
+            return true;
+        }
+
+        std::tuple<winrt::com_ptr<MockControlSettings>, winrt::com_ptr<MockConnection>> _createSettingsAndConnection()
+        {
+            Log::Comment(L"Create settings object");
+            winrt::com_ptr<MockControlSettings> settings;
+            settings.attach(new MockControlSettings());

```

Comment:
derp those are in this namespace, that'll work. I need the `attach` business for the `ControlCore`, etc, since that's in `Control::`, but yea that's way easier

Reply:
wait why the heck do I need that for the remoting tests at all...